app/controllers/dashboard/protocols_controller.rb

```diff
@@ -205,12 +205,6 @@ def display_requests
     render json: data
   end
 
-  def view_details
```

Comment:
I think there was a :view_details symbol in one of the before_filters in this controller. `before_filter find_protocol`. Can you remove it?